test/coverage/communication/RpcServerTest.cpp

```diff
@@ -9,32 +9,400 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
+#include <google/protobuf/util/message_differencer.h>
 #include <gtest/gtest.h>
 #include <up-cpp/communication/RpcServer.h>
 
+#include <memory>
+#include <random>
+#include <sstream>
+
 #include "UTransportMock.h"
 
+namespace uprotocol::test {
+uprotocol::test::UTransportMockInfo UTransportMock::mock_info =
+    uprotocol::test::UTransportMockInfo();
+};
+
+auto& mock_info = uprotocol::test::UTransportMock::mock_info;
```

Comment:
This will be removed when you rebase to main

Reply:
addressed